controllers/login.js

```diff
@@ -47,7 +47,6 @@ router.post('/login/', (req, res, next) => {
 		schoolId,
 		redirect,
 	} = req.body;
-	const validRedirect = redirectHelper.getValidRedirect(redirect);
```

Comment:
is this removed on purpose? (i know its not because we just had a call, so this is just a reminder)